src/app/dynamic_server/AccessControl.h

```diff
@@ -0,0 +1,22 @@
+/**
+ *    Copyright (c) 2022-2023 Project CHIP Authors
+ *
+ *    Licensed under the Apache License, Version 2.0 (the "License");
+ *    you may not use this file except in compliance with the License.
+ *    You may obtain a copy of the License at
+ *
+ *        http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *    Unless required by applicable law or agreed to in writing, software
+ *    distributed under the License is distributed on an "AS IS" BASIS,
+ *    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *    See the License for the specific language governing permissions and
+ *    limitations under the License.
+ */
+#pragma once
+
+/**
+ * Initialize the access control module. Must be called on the Matter task
+ * queue.
+ */
+void initAccessControl();
```

Comment:
Why is this named weirdly (lowercase) and in the global namespace?

Reply:
updated, thanks